locales/en-US.json

```
@@ -18,6 +18,7 @@
   "serverAddress": "Server address",
   "username": "Username",
   "password": "Password",
+  "playType": "Play {playType}",
```

```suggestion
  "playType": "Play {mediaType}",
```

Makes more sense for translators.